src/spatialdata/_core/spatialdata.py

```diff
@@ -315,6 +320,52 @@ def get_instance_key_column(table: AnnData) -> pd.Series:
             return table.obs[instance_key]
         raise KeyError(f"{instance_key} is set as instance key column. However the column is not found in table.obs.")
 
+    @staticmethod
```

Comment:
Three comments:
- [ ] I would move this function to `spatialdata.models._utils`, because `get_channels()` is there
- [ ] I would rename this function to `set_channels()` to be consistent with the old naming
- [ ] Please also update the `api.md` file and the `changelog` file

Reply:
Actually I like the name `set_channel_names()` more than `set_channels()`, so 
- [ ] I would rename `get_channels()` to `get_channel_names()`.